install/customize_pcie_hardware.md

```diff
@@ -0,0 +1,140 @@
+# Customize PCIe Hardware
+
+This page will assist an admin with changing the NCN udev rules for varying PCIe hardware.
+
+> If a system's hardware is Plan of Record then this page is not needed.
+
+## Steps
+
+We'll identify the hardware configuration by PXE booting a node.
+
+1. Shim the netboot, ensuring only the PXE binary is consumed by removing the links generated by `set-sqfs-links.sh`
+
+    ```bash
+    pit# rm /var/www/ncn-*/{initrd.img.xz,kernel,filesystem.squashfs}
+    ```
+
+    The NCNs will fetch the iPXE binary and then pause, this pause prevents the NCN from continuing to boot while we collect information from it.
```

Comment:
"iPXE binary" is used here, but in the main step text, it is "PXE binary". Should this be consistent?